.travis.yml

```diff
@@ -35,7 +28,9 @@ python:
 - 3.4
 
 before_install:
-    - sudo apt-get update; true
+    - ccache -s
+    - pip install --upgrade pip
+    - export PATH=/usr/lib/ccache:${PATH}
```

Comment:
I checked the travis-ci documentation we do not need to do this http://docs.travis-ci.com/user/caching/#ccache-cache